src/darker/__main__.py

```diff
@@ -233,7 +249,7 @@ def main(argv: List[str] = None) -> int:
     failures_on_modified_lines = False
 
     revrange = RevisionRange.parse(args.revision)
-    write_modified_files = not args.check and not args.diff
+    write_modified_files = not args.check and not args.diff and not args.stdout
```

Comment:
Somewhere around here, should there be a check that `len(paths) == 1` when `args.stdout` is truthy?